src/data_sets.h

```diff
@@ -1985,12 +2008,24 @@ typedef struct PACKED
 
 } wheel_config_t;
```

Comment:
If ground_vehicle_t replaced wheel_config_t, do we still need wheel_config_t? Does it have any use/purpose anywhere else in the code?

Reply:
wheel_config_t is used in the nvm_flash_config_t and also inside of ground_vehicle_t.  So we need both.